index.js

```diff
@@ -0,0 +1,35 @@
+const TYPES = ['request', 'response'];
```

Comment:
[PETTY] I would const it, and re-use it in your spec

Reply:
I'll do you one better and remove these strings altogether: ab37205

Reply:
supreme.